src/components/Dropdown/Dropdown.tsx

```diff
@@ -162,20 +183,42 @@ function useFocusContextInstance(
   lastTypedOrSubmittedValue: string,
   setValue: (newValue: string) => void,
   screenReaderKey: number,
-  setScreenReaderKey: (newKey: number) => void
+  setScreenReaderKey: (newKey: number) => void,
+  alwaysSelectOption: boolean,
 ): FocusContextType {
   const [focusedIndex, setFocusedIndex] = useState(-1);
   const [focusedValue, setFocusedValue] = useState<string | null>(null);
   const [focusedItemData, setFocusedItemData] = useState<Record<string, unknown> | undefined>(undefined);
+  useEffect(() => {
+    if (alwaysSelectOption) {
+      if (items.length > 0) {
+        const index = focusedIndex === -1 || focusedIndex >= items.length ? 0 : focusedIndex;
+        setFocusedIndex(index);
+        setFocusedValue(items[index].value);
+        setFocusedItemData(items[index].itemData);
+      } else {
+        setFocusedIndex(-1);
+        setFocusedValue(null);
+        setFocusedItemData(undefined);
+      }
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Could we remove this eslint-disable?

Reply:
We don't want to "lie" to react about deps (there's a good number of articles on this that just parrot Dan Abramov's twitter)

Reply:
ooh yup i had it there when i was experimenting w some stuff lol